### PR TITLE
feat(types): add Context interface

### PR DESCRIPTION
**Pull request by mvrlin**

### CODE REVIEW

**Comment by kevinmarrec** [comment]
We'll need to move types to a `types` folder to follow same practices as I said in https://github.com/nuxt-community/moment-module/pull/17

**Comment by manniL** [comment]
@kevinmarrec Does it fit now? :relaxed:

File: package.json [hunk 1]
```diff
@@ -20,7 +20,9 @@
   },
   "files": [
     "lib",
-    "index.d.ts"
+    "src",
```
Comment (kevinmarrec) [inline package.json]: `src` / `dist` are not needed

**Comment by manniL** [comment]
follow-up in #208 as it seems :man_shrugging:

**Pull request closed without merging**